src/app/GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -561,10 +561,20 @@ protected override void OnActivated(EventArgs e)
                         new WindowsThumbnailToolbarButton(_closeAll.Text, Images.DeleteFile, (s, e) => NativeMethods.PostMessageW(NativeMethods.HWND_BROADCAST, _closeAllMessage))));
             }
 
+            _windowsJumpListManager.EnableThumbnailToolbar(_dashboard?.Visible is not true && Module.IsValidGitWorkingDir());
+
             this.InvokeAndForget(OnActivate);
             base.OnActivated(e);
         }
 
+        protected override void OnDeactivate(EventArgs e)
+        {
+            bool applicationNotActivated = ActiveForm is null;
```

Comment:
I would prefer to use applicationActivated (even if always negated)

Reply:
also renamed